commit_screenshots.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+# Copyright 2019 D.W.J. Bosman
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#      http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# 
+
+set -e # Exit with a non zero exit code when a command fails
+
+SOURCE_BRANCH="master"
+# the screenshots will be committed in this branch
+TARGET_BRANCH="bugs"
+
+# Add the GITHUB_TOKEN to the url so that travis is able to commit
+REPO=`git config remote.origin.url`
+AUTH_REPO=${REPO/https:\/\/github.com\//https://${GITHUB_TOKEN}@github.com/}
+SHA=`git rev-parse --verify HEAD`
+
+git config user.name "Travis CI"
+git config user.email "$COMMIT_AUTHOR_EMAIL"
+
+git fetch --depth 1 origin $TARGET_BRANCH:$TARGET_BRANCH
+git checkout $TARGET_BRANCH
+
+#empty the branch, ignore error if alrady empty
+git rm -r . || true
+
+#add all png files
```

Comment:
[minor] there are a few comments throughout the PR where we include spaces at the beginning, and others where we do not, can we be consistent throughout and include a space? (ex: `# add all...` instead of `#add all`).

Reply:
ok